core/opds2_import.py

```diff
@@ -135,6 +147,53 @@ def parse_manifest(
         return result
 
 
+class PalaceOPDS2PresentationMetadata(opds2_ast.PresentationMetadata):  # type: ignore[misc]
```

Comment:
Minor: I wonder if this code would be better suited to live in the `webpub-manifest-parser` codebase, instead of extending it here. Even if it continues to be a separate class.

Reply:
I had thought the same thing when I realised what needed to be extended, however, since the "timeTracking' flag is Palace specific I thought it should stay within the CM codebase.
This also allows for easier future extensions in the protocol. 
The changes could go in either place though :shrug: